src/components/new-safe/CreateSafe/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Typography, Grid } from '@mui/material'
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft'
+import { useRouter } from 'next/router'
+
+import WalletInfo from '@/components/common/WalletInfo'
+import { useCurrentChain } from '@/hooks/useChains'
+import useWallet from '@/hooks/wallets/useWallet'
+import OverviewWidget from '../OverviewWidget'
+import CreateSafeStep1 from '../steps/Step1'
+
+const CreateSafe = () => {
+  const router = useRouter()
+
+  // TODO: These rows are just a demo
+  const wallet = useWallet()
+  const chain = useCurrentChain()
+  const rows = [
+    ...(wallet && chain ? [{ title: 'Wallet', component: <WalletInfo wallet={wallet} chain={chain} /> }] : []),
+  ]
+
+  const onBack = () => {
+    router.back()
+
+    // Logic to be handled by stepper hook
+  }
+
+  // TODO: Improve layout when other widget/responsive design is ready
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={1} />
```

Comment:
I would suggest not adding these offsets here. From the designs it looks like the form will be centered and the sidebar will be hidden.